engine/src/main/java/org/hibernate/validator/internal/util/scriptengine/DefaultLookupScriptEvaluatorFactory.java

```diff
@@ -29,11 +26,6 @@
  */
 public class DefaultLookupScriptEvaluatorFactory extends AbstractCacheableScriptEvaluatorFactory {
 
-	/**
```

Comment:
@marko-bekhta ah have a misunderstanding here. I was asking to remove the `SoftReference` but keep the singleton instance, just not with a `SoftReference` :).

Reply:
right ... It looks like that that's how I addressed this ticket - https://hibernate.atlassian.net/browse/HV-1469 
And then when I've added that part with the external classloader we cannot cache the instance unless it's just one external loader (and it's passed to `getInstance()`), right?